SocialCareCaseViewerApi/V1/Gateways/DatabaseGateway.cs

```diff
@@ -735,7 +735,8 @@ public PostWarningNoteResponse PostWarningNote(PostWarningNoteRequest request)
                 Note = $"{dt.ToShortDateString()} | Warning Note | Warning note created against this person",
                 FormNameOverall = "API_WarningNote",
                 FormName = "Warning Note Created",
-                WarningNoteId = warningNote.Id.ToString()
+                WarningNoteId = warningNote.Id.ToString(),
+                WorkerEmail = request.CreatedBy
```

Comment:
Do we have tests for this? Could we add a test?

Reply:
The test is in the `DatabaseGateWayTests` under `PostWarningNoteShouldCallInsertCaseNoteMethod`.

Reply:
I'm inclined to think we do need a test or a change of an existing test here.

My thought is if we undid the code change you made would any test fail? Ideally our tests can help stop us introducing bugs or any regressions in the future.

Reply:
@JohnFarrellDev There was a brief convo about the testing with the MongoDB [here](https://hackit-lbh.slack.com/archives/C021Q88C953/p1620914341030000), but I agree that we should give testing this a try if we can.